src/Sulu/Bundle/AdminBundle/Resources/js/components/Navigation/Navigation.js

```diff
@@ -122,76 +109,45 @@ class Navigation extends React.Component<Props> {
-            title,
             username,
+            userImage,
             onLogoutClick,
             onProfileClick,
             suluVersion,
             suluVersionLink,
             onPinToggle,
         } = this.props;
 
-        const pinClass = classNames(navigationStyles.pin, {[navigationStyles.pinActive]: pinned});
+        const pinClass = classNames(navigationStyles.pin, {[navigationStyles.active]: pinned});
 
         return (
             <div className={navigationStyles.navigation}>
                 <div className={navigationStyles.header}>
-                    <div className={navigationStyles.headerContent}>
-                        <Icon className={navigationStyles.headerIcon} name="su-sulu" />
-                        <span className={navigationStyles.headerTitle}>{title}</span>
-                    </div>
-                </div>
-                <div className={navigationStyles.user}>
-                    <div className={navigationStyles.userContent}>
-                        {this.renderUserImage()}
-                        <div className={navigationStyles.userProfile}>
-                            <button className={navigationStyles.username} onClick={onProfileClick}>{username}</button>
-                            <button
-                                className={navigationStyles.logout}
-                                onClick={onLogoutClick}
-                            >
-                                <Icon name="su-exit" />
-                                Log out
-                            </button>
+                    <Icon className={navigationStyles.logo} name="su-sulu-logo" />
+
+                    {onPinToggle &&
+                        <div className={pinClass} onClick={this.handlePinToggle} role="button">
+                            <Icon className={navigationStyles.pinIcon} name="su-stick-right" />
                         </div>
-                    </div>
+                    }
                 </div>
+
                 <div className={navigationStyles.items}>
                     {this.cloneChildren()}
                 </div>
+
                 <div className={navigationStyles.footer}>
-                    {onPinToggle &&
-                        <button className={pinClass} onClick={this.handlePinToggle} type="button">
-                            <Icon className={navigationStyles.pinIcon} name="fa-thumb-tack" />
-                        </button>
-                    }
-                    <div className={navigationStyles.versions}>
-                        {this.renderAppVersion()}
-                        <div>
-                            Sulu (<a href={suluVersionLink} rel="noopener noreferrer" target="_blank">{suluVersion}</a>)
-                        </div>
-                    </div>
+                    <ProfileButton
+                        onLogoutClick={onLogoutClick}
+                        onProfileClick={onProfileClick}
+                        suluVersion={suluVersion}
```

Comment:
think its a bit strange that the version is displayed between user-related actions in the design. think i would display it above the accordion personally. would like to talk about this with the rest of the team 🙂